ui/static/ui/js/listing_resources.jsx

```diff
@@ -132,7 +132,7 @@ define('listing_resources', ['react', 'jquery', 'lodash'],
                     </span>
                     <span className="meta-item mi-col-3">
                       <i className="fa fa-graduation-cap">
-                      </i> {resource.xa_avg_grade}
+                      </i> {resource.xa_avg_grade.toFixed(1)}
```

Comment:
It outputs the number formatted to that many decimal places. For example:

   > 0.3.toFixed(3)
   '0.300'

It's only here to match Django's default behavior when dealing with floats without a decimal component.